pkg/profiler/cpu/cpu.go

```diff
@@ -886,6 +982,14 @@ func (p *CPU) obtainRawData(ctx context.Context) (profile.RawData, error) {
 			}
 		}
 
+		if key.InterpreterStackID != 0 {
+			// TODO: Improve error handling.
+			interpErr := p.bpfMaps.readInterpreterStack(key.InterpreterStackID, interpreterStack)
```

Comment:
I think this method should return the actually read symbol table from BPF same, and `obtainRawData` should return it to the caller. It makes more sense to follow.